[Next.js] Better Vercel deployment (#6441)
plone/volto: apps/nextjs/README.md

API_SERVER_URL=https://my-server-DNS-name.tld/api
```

> For production deployments, we need to force the deployment URL, otherwise, we have CORS problems.

Reviewer: ```suggestion
For production deployments, you need to force the deployment URL, otherwise you will have issues with CORS.
```

> We need another environment variable to set the production URL: `NEXT_PRODUCTION_URL`.

Reviewer: ```suggestion
To do so, set another environment variable for the production URL, `NEXT_PRODUCTION_URL`.
```

> This URL needs to be schema-less, so without `http` or `https`:

Reviewer: ```suggestion
This URL needs to be scheme-less, without `http` or `https`, and consist only of the domain name:
```
